### PR TITLE
[tplinksmarthome] Document sending raw commands to devices

### PR DESCRIPTION
**Pull request by timcharper**
Signed-off-by: Tim Harper <timcharper@gmail.com>

### CODE REVIEW

**Comment by timcharper** [comment]
https://community.openhab.org/t/tplink-plugin-turns-on-dimmer-when-sending-brightness/142454/8

File: bundles/org.openhab.binding.tplinksmarthome/README.md [hunk 1]
```diff
@@ -439,3 +439,22 @@ Color        TP_LB_Color  "Color"                     <slider> { channel="tplink
 Switch       TP_LB_ColorS "Switch"                             { channel="tplinksmarthome:lb130:bulb2:color" }
 Switch       TP_O_OUTLET  "Outdoor Outlet"                     { channel="tplinksmarthome:kp401:outlet:switch" }
 ```
+
+## Sending Raw Commands to Devices (Advanced Usage)
+
+TPLinkSmarthome Things can be sent a raw JSON string to control a device in a way not directly supported by this binding. You can find several JSON commands in the [test fixtures](https://github.com/openhab/openhab-addons/tree/main/bundles/org.openhab.binding.tplinksmarthome/src/test/resources/org/openhab/binding/tplinksmarthome/internal/model) for this binding.
```
Comment (fwolter) [inline bundles/org.openhab.binding.tplinksmarthome/README.md:445]: If you put each sentence in a separate line, reading the diffs would be easier.

File: bundles/org.openhab.binding.tplinksmarthome/README.md [hunk 2]
```diff
@@ -439,3 +439,22 @@ Color        TP_LB_Color  "Color"                     <slider> { channel="tplink
...
+
+As an example, you might want to change the brightness level of a dimmer without turning it on or off. Given the dimmer Thing has an id of `tplinksmarthome:hs220:123ABC`, you could accomplish just that with the following rule:
+
+`example.rules`
+
+```
```
Comment (fwolter) [inline bundles/org.openhab.binding.tplinksmarthome/README.md:451]:
```suggestion
```java
```

**Review by fwolter — CHANGES_REQUESTED** [review]

**Review by fwolter — APPROVED** [review]
LGTM

**Pull request merged**